recipes-bsp/u-boot/u-boot-toradex_%.bbappend

```diff
@@ -0,0 +1,11 @@
+COMPATIBLE_MACHINE = "(colibri-imx7|colibri-imx7-mender)"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+# Add Patch for selecting correct resolution for 7" display
+
+SRC_URI_append_colibri-imx7 = " \
```

Comment:
What did we decide about colibri-imx7 vs colibri-imx7-mender?  Do we still have both?

Reply:
I'm not aware of any reason to keep colibri-imx7-mender, but I wasn't involved in the initial port. I'm in favor of removing it, but right now this repository is just following what we have, so I wouldn't make this dependent on that work.